packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -1046,8 +1046,8 @@ function propsDiffer(src: Record<string, any>, dst: Record<string, any>): boolea
   if (!src || !dst) {
     return true;
   }
-  let srcKeys = removeChildrenKey(Object.keys(src));
```

Comment:
in props we don't have children anymore, so I assume this is not needed now. Let me know if I am wrong

Reply:
Right, the optimizer makes sure combined with _jsxC which checks at runtime. 

However, the children prop can still be in the varProps, it should just be ignored